### ISSUE TITLE
Add mobile support

### ISSUE THREAD

**Issue by Wildhoney** [Wildhoney/Leaflet.FreeDraw#3]

**Comment by Wildhoney** [comment]
@pztrick Did you have any luck in supporting mobile?

**Comment by pztrick** [comment]
@Wildhoney No, I never made the time, unfortunately.

**Comment by nmccready** [comment]
Anyway some peeps can work together?  https://github.com/Leaflet/Leaflet.draw/pull/285

**Comment by pztrick** [comment]
@nmccready: As someone that uses the Leaflet.draw fork that added mobile support, I definitely think it would be cool for these two projects to figure out mobile together. It's been over a year since I implemented touch in my last project and I'm much too time-poor to be the one that dives in unfortunately. :(

So... here is my :+1:.

**Comment by nmccready** [comment]
Yeah I know how that goes. Well by looking at the above ticket.. I need to pound out a 1.X leaflet branch on [angular-leaflet-directive](https://github.com/tombatossals/angular-leaflet-directive/)

**Comment by Yago** [comment]
@Wildhoney you are my hero 😍  !!!

Last week, I did a lot of research and tests about map free draw on touch devices and I was so sad to not find a good solution. Even yours, the far better Leaflet (that I love) module doesn't... I come back today and see the v2 release which support touch devices pretty damn well :scream: Perfect timing ! 

Our client don't want to negotiate that feature, you totally save my life. Thank you so much for your work !

**Comment by Wildhoney** [comment]
Hahaha... That's good! You're more than welcome 👍 

I think it still needs some work for mobiles, but at least it's better than how it was.

**Comment by GeoTob** [comment]
After reading through this issue and particularly @Yago's comment: 

> I come back today and see the v2 release which support touch devices pretty damn well

I'm not sure if touch is supposed to work or not? So far I'm not having any luck when trying to use this on mobile, but might have missed a setting somewhere?

**Comment by BaiShaojie123123** [comment]
Hello friends, recently do a similar function, but I want to use other maps, I am a beginner, hoping to get guidance. Mainly want to know the painting and get the painting after the corner of the problem, hope to get your Thanks for the help

**Comment by Wildhoney** [comment]
@BaiShaojie123123 I'm not sure what you're asking. Please clarify?

**Comment by dgautsch** [comment]
Is mobile still unsupported? I can't seem to get it to work.

**Comment by ineeve** [comment]
It does not seem to work on mobile yet ...

**Comment by lukszi** [comment]
As a workaround I tried to use [https://stackoverflow.com/questions/1517924/javascript-mapping-touch-events-to-mouse-events](this), but this solution has some problems
- using deprecated methods
- touchMove triggering mouseMove and scrolling
- blocking out my leaflet-controls

limiting the above solution to only the viewport didn't do any good to unlock the buttons either.

**Comment by alessiogaronzi** [comment]
@lukszi Hi your workaround work for mobile in Chrome?

**Comment by yellerkaleb** [comment]
Does anyone have a solution or alternative to this issue?

**Comment by yellerkaleb** [comment]
this works, (you will may have to manually type in the link):

https://stackoverflow.com/questions/1517924/javascript-mapping-touch-events-to-mouse-events

**Comment by FluorescentHallucinogen** [comment]
Looking forward for touch support.

Just found https://github.com/bozdoz/leaflet-freehandshapes (https://bozdoz.github.io/leaflet-freehandshapes/), works on Android like a charm!
